packages/contracts/contracts/interfaces/v0.8.x/ISharedMinterDALinV0.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+pragma solidity ^0.8.0;
+
+interface ISharedMinterDALinV0 {
```

Comment:
types for events in this library should be updated to match the function inputs and storage (e.g. use `uint64` instead of `uint256`) - this will save on gas costs by reducing quantity of bytes in log data, as well as avoiding a cast to uint256 🥳